CommunityToolkit.Tooling.SampleGen/ToolkitSampleMetadataGenerator.Sample.cs

```diff
@@ -163,10 +163,6 @@ void Execute(IncrementalValuesProvider<ISymbol> types, bool skipDiagnostics = fa
 
     private static void CreateSampleRegistry(SourceProductionContext ctx, Dictionary<string, ToolkitSampleRecord> sampleMetadata)
     {
-        // TODO: Emit a better error that no samples are here?
```

Comment:
I'll file a ticket for this, but let's discuss a little first.

We don't want to use a warning since the CI will turn that into an error in release mode. Emitting an info diagnostic would be fine.

@michael-hawker Do we remember why we added this? We decided that we want things to compile even with missing samples and components - do we still need this at compile time? Maybe a UI indicator at time would work for our purposes.

Reply:
Yeah, we can open a ticket and discuss. I'd be surprised if we have docs but without samples, so it's mainly just an indicator to help guide folks on how to make/add samples if none are contained in the document. So an info diagnostic may be the approach.

Reply:
Those should be covered by [`DiagnosticDescriptors.SampleNotReferencedInMarkdown`](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/blob/a51f9035c40a606fdadfccbee7718e0080c999db/CommunityToolkit.Tooling.SampleGen/Diagnostics/DiagnosticDescriptors.cs#L188) and [`DiagnosticDescriptor.DocumentationHasNoSamples`](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/blob/a51f9035c40a606fdadfccbee7718e0080c999db/CommunityToolkit.Tooling.SampleGen/Diagnostics/DiagnosticDescriptors.cs#L200).